### DISCUSSION TITLE
TradeMe: Lands compatibility.

### DISCUSSION THREAD

**Discussion by MCTGE** [Zrips/CMI#6800] [Feature Requests]
It would be amazing if[ Lands ](https://www.spigotmc.org/resources/lands-land-claim-plugin-grief-prevention-protection-gui-management-nations-wars-1-18-support.53313/) compatibility was added into the plugin :)

**Comment by MrCrazys** [comment]
Not a CMI suggestion.

**Comment by mrfloris** [comment]
> It would be amazing if[ Lands ](https://www.spigotmc.org/resources/lands-land-claim-plugin-grief-prevention-protection-gui-management-nations-wars-1-18-support.53313/) compatibility was added into the plugin :)

Bug reports and feature requests:
https://github.com/Zrips/TradeMeAddon/issues

You can post TradeMe feature-requests in the other repository. ^_^
